docs/ftldns/dns-cache.md

```diff
@@ -0,0 +1,22 @@
+`pihole-FTL` offers an efficient DNS cache that helps speeding up your Internet experience. Setting the cache size to zero disables caching. The DNS TTL value is used for determining the caching period. `pihole-FTL` clears its cache on receiving `SIGHUP`.
```

Comment:
Perhaps note that the DNS cache is part of dnsmasq, so people know that it should work in the same/similar way?